torch/_export/serde/serialize.py

```diff
@@ -1496,56 +1631,10 @@ def deserialize_sym_int(self, s: SymInt) -> Union[int, torch.SymInt]:
             else:
                 assert val.hint.type == "as_int"
                 hint = val.hint.value
-
-            if val.expr_str in self.symbol_name_to_symbol:
-                sym = self.symbol_name_to_symbol[val.expr_str]
+            if val.expr_str != "":
```

Comment:
Would it be better to check like, if hasattr(val, "expr_ast")?

Reply:
Ah the field isn't optional, and it's BC breaking if we change that. I could switch to None instead.